### PR TITLE
style(): change all private identifiers to leading underscore

### PR DESCRIPTION
**Pull request by jelbourn**
R: @traviskaufman

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "no-bitwise": true,
     "no-shadowed-variable": true,
     "no-unused-expression": true,
-    "no-unused-variable": [true, {"ignore-pattern": "ddescribe|iit|.*_$"}],
+    "no-unused-variable": [true, {"ignore-pattern": "ddescribe|iit|.*_$|^_.*"}],
```
Comment (hansl) [inline tslint.json]: `^(ddescribe|iit|.*_|_.*)$`

**Comment by traviskaufman** [comment]
@hansl has one comment but besides that :fire: :100: LGTM

**Comment by jelbourn** [comment]
@hansl @traviskaufman changed what this PR does. I just did the whole replacement.

Another success story for WebStorm.

Replaced: `([a-zA-Z]+)_([^a-z])`
With: `_$1$2`

**Comment by traviskaufman** [comment]
LGTM!

**Comment by hansl** [comment]
Ship it!

**Pull request merged**